torch_frame/data/multi_nested_tensor.py

```diff
@@ -187,6 +190,25 @@ def _apply(self, fn: Callable[[Tensor], Tensor]) -> 'MultiNestedTensor':
 
         return out
 
+    def dim(self) -> int:
+        return 2
+
+    def size(self, dim) -> int:
+        if dim == 0:
+            return self.num_rows
+        elif dim == 1:
+            return self.num_cols
+        else:
+            raise ValueError(f"Dim {dim} does not exist.")
+
+    @staticmethod
+    def stack(xs: List['MultiNestedTensor'],
+              dim: int = 0) -> 'MultiNestedTensor':
```

Comment:
how does it depend on `dim`?

Reply:
Can you add a TODO so that I can properly handle this once I added concatenation functionality?

Reply:
Sure!

Reply:
btw: This is not a stack but a cat :)

Reply:
Noted. I was planning to modify it once `MultiNestedTensor` is in.